### ISSUE TITLE
Imports not generated for errors in an operation that have been imported from another namespace

### ISSUE THREAD

**Issue by Pete1232** [disneystreaming/smithy4s#100]
Using Smithy `0.12.0`. Example:

`errors.smithy`
```
namespace com.example.smithy.errors

@error("client")
@httpError(404)
structure NotFoundError {
  error: String,
}
```
`example.smithy`
```
namespace com.example.smithy

service ExampleService {
  operations: [ExampleOperation],
  errors: [com.example.smithy.errors#NotFoundError],
}

@http(method: "GET", uri: "/example", code: 200)
operation ExampleOperation {
  input: ExampleInput,
  output: ExampleOutput,
  errors: [com.example.smithy.errors#NotFoundError],
}

@input
structure ExampleInput {
  @httpQuery("id")
  id: String
}

@output
structure ExampleOutput {
  id: String
}
```

The generated `ExampleService.scala` makes reference to the `NotFoundError`, but it isn't included in the class imports.

**Comment by Pete1232** [comment]
I actually originally noticed a different issue, where referencing an error in a different namespace resulted in a redundant package import, but I found this trying to reproduce it.

**Comment by Pete1232** [comment]
This seems to be fixed after upgrading to `0.12.1`. I'll close this and try to reproduce the other error I mentioned
